.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+**Description**
+A description of the PR, should include a decent explanation as to why this change was needed and a decent explanation as to what this change does
+
+**Review Instructions**
+Describe if this ticket needs review and if so, how one may go about it in qa and/or staging environments.
+For example, a ticket based on Security Hub, Snyk, or Dependabot may not need review since those services 
+will generate new warnings if the issue has not been resolved properly. On the other hand, an infrastructure
+ticket that results in visible changes to the end-user will definitely require review. 
+Many tickets will likely be between these two extremes, so some judgement may be required.
+
+**Issue**
+A link to a github issue or SEAB- ticket (using that as a prefix)
```

Comment:
Did we reach a consensus on whether we should include **both** the github and DOCK issue for tickets instead of just one?

If so I think we should note that here.

Reply:
Not exactly,I think this is my preference though (github isssue for public tickets, SEAB for not)

Reply:
Sweet! Sounds good